src/components/Tool/Appeal/List/ContactRow/ContactRow.tsx

```diff
@@ -134,184 +133,194 @@ export const ContactRow: React.FC<Props> = ({
 
   return (
     <>
-      <ListButton
-        focusRipple
-        onClick={handleContactClick}
-        onMouseEnter={preloadContactsRightPanel}
-        className={clsx({
-          'top-margin': useTopMargin,
-          checked: isChecked(contactId),
-        })}
-        data-testid="rowButton"
+      <NextLink
+        href={contactUrl}
+        scroll={false}
+        prefetch={false}
+        shallow
+        legacyBehavior
+        passHref
+        style={{ width: '100%', color: 'initial' }}
+        data-testid="contactRowLink"
       >
-        <Grid container alignItems="center">
-          <Grid
-            item
-            xs={isExcludedContact ? 5 : 6}
-            style={{ paddingRight: 16 }}
-            display={'flex'}
-          >
-            <Hidden xsDown>
-              <ListItemIcon>
-                <StyledCheckbox
-                  checked={isChecked(contact.id)}
-                  color="secondary"
-                  onClick={(event) => event.stopPropagation()}
-                  onChange={() => onContactCheckToggle(contact.id)}
-                  value={isChecked}
-                />
-              </ListItemIcon>
-            </Hidden>
-            <ListItemText
-              primary={
-                <Typography component="span" variant="h6" noWrap>
-                  <Box component="span" display="flex" alignItems="center">
-                    {name}
+        <ListButton
+          LinkComponent={'a'}
```

Comment:
Same question of whether this does anything without an `href` prop and since I thought the default link component is already `a`.